tests/cli.js

```diff
@@ -143,6 +143,48 @@ describe('cli', function () {
     });
   });
 
+  // Test default config files
+
+  it('should return JSON from .sass-lint.yml', function (done) {
+    var command = 'sass-lint ../../sass/cli.scss --verbose';
+
+    childProcess.exec(command, { cwd: path.join(__dirname, 'yml', '.sass-lint.yml') }, function (err, stdout) {
```

Comment:
I personally find it confusing when a folder is named exactly like a file. Your intention for adding those directories might be clearer if you grouped them in a descriptively-named parent folder. For example:
```
/yml/root-directories/
    sass-lint-yml/.sass-lint.yml
    sasslintrc/.sasslintrc
```
 `root-directories` clarifies why additional directories are used. If others think it is clearer to use the current directory scheme I'm fine with that.